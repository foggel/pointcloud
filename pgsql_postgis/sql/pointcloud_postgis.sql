CREATE SCHEMA pp;
CREATE EXTENSION "pointcloud_postgis" SCHEMA pp CASCADE;
