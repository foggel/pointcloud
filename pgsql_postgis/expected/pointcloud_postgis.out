CREATE SCHEMA pp;
CREATE EXTENSION "pointcloud_postgis" SCHEMA pp CASCADE;
NOTICE:  installing required extension "postgis"
NOTICE:  installing required extension "pointcloud"
